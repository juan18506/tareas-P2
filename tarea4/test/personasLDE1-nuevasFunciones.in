crearListaPersonas

# creo 8 personas y las agrego a la lista

crearPersona 1 11 Persona1
insertarPersonaListaPersonas 1

crearPersona 2 22 Persona2
insertarPersonaListaPersonas 2

crearPersona 3 33 Persona3
insertarPersonaListaPersonas 3

crearPersona 4 44 Persona4
insertarPersonaListaPersonas 4

crearPersona 5 55 Persona5
insertarPersonaListaPersonas 5

crearPersona 6 66 Persona6
insertarPersonaListaPersonas 6

crearPersona 7 77 Persona7
insertarPersonaListaPersonas 7

crearPersona 8 88 Persona8
insertarPersonaListaPersonas 8

# imprimo la cantidad y la lista

imprimirCantidadListaPersonas
imprimirListaPersonas

# verifico que esten 4 personas distintas y las obtengo 

estaPersonaConNombreEnTPersonasLDE Persona1
obtenerPersonaConNombreTPersonasLDE Persona1

estaPersonaConNombreEnTPersonasLDE Persona3
obtenerPersonaConNombreTPersonasLDE Persona3

estaPersonaConNombreEnTPersonasLDE Persona6
obtenerPersonaConNombreTPersonasLDE Persona6

estaPersonaConNombreEnTPersonasLDE Persona8
obtenerPersonaConNombreTPersonasLDE Persona8

# ahora elimino a esas personas y verifico que no estan mas

eliminarPersonaConNombreTPersonasLDE Persona1
estaPersonaConNombreEnTPersonasLDE Persona1

eliminarPersonaConNombreTPersonasLDE Persona3
estaPersonaConNombreEnTPersonasLDE Persona3

eliminarPersonaConNombreTPersonasLDE Persona6
estaPersonaConNombreEnTPersonasLDE Persona6

eliminarPersonaConNombreTPersonasLDE Persona8
estaPersonaConNombreEnTPersonasLDE Persona8

# imprimo la cantidad y la lista

imprimirCantidadListaPersonas
imprimirListaPersonas

# libero la lista
liberarListaPersonas

Fin