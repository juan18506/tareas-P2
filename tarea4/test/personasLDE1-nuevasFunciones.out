1>2>#  creo 8 personas y las agrego a la lista.
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>#  imprimo la cantidad y la lista.
20>La cantidad de personas en la lista es: 8.
21>Persona 1: Persona1, 11 años
Persona 2: Persona2, 22 años
Persona 3: Persona3, 33 años
Persona 4: Persona4, 44 años
Persona 5: Persona5, 55 años
Persona 6: Persona6, 66 años
Persona 7: Persona7, 77 años
Persona 8: Persona8, 88 años
22>#  verifico que esten 4 personas distintas y las obtengo .
23>La persona con nombre Persona1 está en la lista.
24>Persona 1: Persona1, 11 años
25>La persona con nombre Persona3 está en la lista.
26>Persona 3: Persona3, 33 años
27>La persona con nombre Persona6 está en la lista.
28>Persona 6: Persona6, 66 años
29>La persona con nombre Persona8 está en la lista.
30>Persona 8: Persona8, 88 años
31>#  ahora elimino a esas personas y verifico que no estan mas.
32>33>La persona con nombre Persona1 NO está en la lista.
34>35>La persona con nombre Persona3 NO está en la lista.
36>37>La persona con nombre Persona6 NO está en la lista.
38>39>La persona con nombre Persona8 NO está en la lista.
40>#  imprimo la cantidad y la lista.
41>La cantidad de personas en la lista es: 4.
42>Persona 2: Persona2, 22 años
Persona 4: Persona4, 44 años
Persona 5: Persona5, 55 años
Persona 7: Persona7, 77 años
43>#  libero la lista.
44>45>Fin.
